qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/project/datamodel/qute-resolvers.json

```diff
@@ -1,7 +1,7 @@
 {
 	"resolvers": [
 		{
-			"signature": "orEmpty(arg : java.lang.Iterable<T>) : java.util.List<T>",
+			"signature": "orEmpty(self : java.lang.Iterable<T>) : java.util.List<T>",

```

Comment:
please use `base `instead of `self`:

"signature": "orEmpty(base : java.lang.Iterable<T>) : java.util.List<T>"